qualtrics_link/util.py

```diff
@@ -226,10 +226,13 @@ def filter_person_list(person_list):
             employee_list.append(person)
 
     # If more than one Person with Employee role, check if any have prime indicator set to 'Y'
+    # Else return the first employee record
     if len(employee_list) > 0:
```

Comment:
If there is one record returned, we check if the prime role is set to 'Y' if not we are still returning the first item in that list.
len(list('a')) will return 1 not 0